content/03.categorize.md

```diff
@@ -231,6 +231,58 @@ a need for better pathologist-computer interaction techniques that will allow
+We categorize the body of work by the application domain/text genre (biomedical literature vs. clinical notes) and by the actual task (e.g. concept or relation extraction).
+
+In biomedical text mining, named entity recognition (NER) is a task of identifying text spans that refer to a biological concept of a specific class, such as disease or chemical, in a controlled vocabulary or ontology.
+NER is of significant importance as it is often needed as a first step in many complex text mining systems.
+The current state-of-the-art methods typically reformulate the task as a sequence labeling problem and use conditional random fields [@doi:10.1093/bioinformatics/btw343; @doi:10.1093/bioinformatics/btt156; @doi:10.1093/bioinformatics/btt474].
+In recent years, word embeddings that contain rich latent semantic information of words have been widely used to improve the NER performance.
+[@doi:10.3390/info6040848] studied the effect of word embeddings on drug name recognition and compared them with traditional semantic features.
+[@doi:10.1155/2014/240403] investigated word embeddings in gene, DNA, and cell line mention detection tasks.
+Moreover, [@doi:10.18653/v1/w15-3822] examined the use of neural word embeddings for clinical abbreviation disambiguation.
+[@doi:10.18653/v1/w15-3810] exploited task-oriented resources to learn word embeddings for clinical abbreviation expansion.
+
+Relation extraction is a task of detecting and classifying semantic relationship between entities from the literature.
+At present, kernel methods or feature-based approaches are commonly applied [@doi:10.1371/journal.pcbi.1000837; @doi:10.1186/s13321-016-0165-z; @doi:10.1093/bioinformatics/btp602].
+There are several proposals of using deep learning in the biomedical relation tasks because it can relieve the feature sparsity and engineering problems.
+Some studies focused on jointly extracting biomedical entities as well as their relations simultaneously [@tag:li2016joint; @doi:10.1186/s12859-017-1609-9], while others applied deep learning on relation classification with given entities.
+For example, [@doi:10.18653/v1/w17-2304] proposed a multichannel dependency-based CNN and [@doi:10.1155/2016/8479587; @doi:10.1155/2016/1850404] proposed a shortest path based CNN for the sentence-based protein-protein extraction task.
+[@doi:10.1504/IJDMB.2016.074878] proposed a biomedical domain-specific word embedding model to reduce the manual labor of designing semantic representation for the same task.
+[@doi:10.1093/database/bax024] employed a maximum entropy model and a CNN model for chemical-induced disease relation extraction at inter- and intra-sentence level, respectively.
+For drug-drug interaction, [@doi:10.1093/bioinformatics/btw486] used a syntax CNN, [@doi:10.18653/v1/w17-2302] used an attention CNN, and [@arxiv:1705.03261] examined a recurrent neural network model with multiple attention layers.
```

Comment:
Can you provide one sentence right after this with a quick comparison of syntax CNN / attention CNN / RNN relative benefits/drawbacks in this task? Also, can you define RNN here right after saying "recurrent neural network"? We're going to have to go through and identify the first use of each term, and having the RNN abbrev will help.